python/ray/_private/worker.py

```diff
@@ -2638,6 +2638,9 @@ def get(
             the get takes longer than timeout to return.
         Exception: An exception is raised if the task that created the object
             or that created one of the objects raised an exception.
+            It's worth noting, if any actor is crashed, and if you do `ray.get`
+            on crashed/uncrashed actors together, an exception won't be raised
+            until uncrashed actors are finished.
```

Comment:
What we want to document is the intended behavior that if you ray.get() multiple objects, if one object is an exception, ray.get() will immediately return without waiting for the remaining ones to finish. You can test it out by

```
import ray
import time

ray.init()

@ray.remote
def task1():
  raise ValueError("fail")

@ray.remote
def task2():
  time.sleep(1000000)


ray.get([task1.remote(), task2.remote()])
```

The PR you referred is actually a bug since it doesn't conform to the intended behavior.

Reply:
Ah gotcha, sorry I thought it's asking to document `known issues` part. Updated comments, thanks for the example!